src/BlazorWebFormsComponents/BaseWebFormsComponent.cs

```diff
@@ -162,6 +159,11 @@ void ParentWrappingBuildRenderTree(RenderTreeBuilder builder)
 		[Parameter]
 		public RenderFragment ChildComponents { get; set; }
 
+		protected virtual void DataBinding(EventArgs e)
+		{
+			OnDataBinding.InvokeAsync(e);
```

Comment:
Does this need a null check, or invoke like this?

```c#
			OnDataBinding?.InvokeAsync(e);
```

Reply:
Also, it's calling an async method. Does this method need to be async, and include an `await`?

Reply:
ouch... you're right...  but to match API from back-in-the-day Web Forms, we should probably wrap the InvokeAsync with .GetAwaiter().GetResult() to force synchronous behavior.